src/binding/transaction_log_file_posix.cpp

```diff
@@ -100,6 +100,12 @@ int64_t TransactionLogFile::readFromFile(void* buffer, uint32_t size, int64_t of
 bool TransactionLogFile::removeFile() {
 	std::unique_lock<std::mutex> lock(this->fileMutex);
 
+	if (this->memoryMap) {
+		DEBUG_LOG("%p TransactionLogFile::removeFile Releasing memory map before removing file: %s\n",
+			this, this->path.string().c_str())
+		this->memoryMap.reset();
```

Comment:
@cb1kenobi I thought this would happen automatically in the destructor (and doesn't that happen right after removing the file)? Curious of why it would need to be done here.